rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -42,27 +43,28 @@ def get_ppo_loss(fwd_in, fwd_out):
     return loss
 
 
+@ExperimentalAPI
 @dataclass
 class PPOModuleConfig(RLModuleConfig):
-    """Configuration for the PPO module.
+    """Configuration for the PPO RLModule.
 
     Attributes:
+        observation_space: The observation space of the environment.
+        action_space: The action space of the environment.
+        shared_encoder_config: The configuration for the encoder network.
         pi_config: The configuration for the policy network.
         vf_config: The configuration for the value network.
-        shared_encoder_config: The configuration for the encoder network.
         free_log_std: For DiagGaussian action distributions, make the second half of
             the model outputs floating bias variables instead of state-dependent. This
             only has an effect is using the default fully connected net.
-        shared_encoder: Whether to share the encoder between the pi and value
     """
 
+    observation_space: gym.Space = None
```

Comment:
These two spaces props are already part of the parent. Do we have to repeat those here?

Reply:
Changed!